addons/l10n_fr_account/models/res_partner.py

```diff
@@ -10,4 +10,6 @@ def _compute_company_registry_placeholder(self):
         for partner in self:
             country = partner.ref_company_ids[:1].account_fiscal_country_id or partner.country_id
             if country.code == 'FR':
-                partner.company_registry_placeholder = '12345678900001'  # SIRET
+                partner.company_registry_placeholder = _('SIRET Number')
+            elif country.code == 'PF':
+                partner.company_registry_placeholder = _('N° Tahiti') # PF adheres to French regulations
```

Comment:
Isn't N° mostly used in French to contract Numero? Like in English, wouldn't people use more No.? (and like Tahiti Numero instead of Numero Tahiti?) 
This can definitely be the French translation but I'm not sure of the base in English